.eslintignore

```diff
@@ -4,6 +4,7 @@ node_modules
 coverage
 /.next
 /next.config.js
+/docs/build
```

Comment:
So the reason I had to do this was because it appeared that this directory was being cached in circleci and was breaking builds. I've pushed 7a54cd01f to see if this is now not an issue.